SERVER/src/checkpoint_manager.py

```
@@ -1,105 +1,115 @@
-            return "Error: Invalid export format. Choose 'text' or 'json'.", None
+import os
+import shutil
+from datetime import datetime
+import json
+from .config import DB_PATH, ADAPTERS_PATH, BASE_CHECKPOINT_PATH, BASE_DATA_PATH # Import from config
+
+# Path for server-specific adapters (Actor1)
+# Now configurable via CheckpointManager arguments for model and Actor ID.
+
+class CheckpointManager:
+    def __init__(self, server_model_name="TinyLLaMA", server_Actor_id="Actor1"):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)